pkg/tnf/handlers/handler_template/handler.tmpl

```diff
@@ -36,61 +31,45 @@ const (
 	SuccessfulOutputRegex = `.+`
 )
 
-// Args returns the command line args for the test.
 func (h *{{ .Handlername }}) Args() []string {
-	return h.args
+	//fill the stub
 }
 
 // GetIdentifier returns the tnf.Test specific identifier.
 func (h *{{ .Handlername }}) GetIdentifier() identifier.Identifier {
-	return identifier.HostnameIdentifier
+	//fill the stub
 }
 
 // Timeout return the timeout for the test.
 func (h *{{ .Handlername }}) Timeout() time.Duration {
-	return h.timeout
+	//fill the stub
```

Comment:
same thing here. The ``timeout`` is common across all the files. You can keep this function in the template